### DISCUSSION TITLE
Getting Pacstall Support In GUI Software Clients

### DISCUSSION THREAD

**Discussion by trymeouteh** [pacstall/pacstall#551] [General]
Many Arch based distros have their default package manager integrated with the AUR were you can use Pamac for example to install and receive updates to packages from the AUR.

Are GUI software clients on Debian, Ubuntu, Mint and other Debian based distros working on integrating Pacstall into their clients and to have the GUI updater clients do updates to Pacstall packages? I understand Pacstall itself cannot make this happen since they do not control projects like GNOME Software.

To my understanding, Pacstall is ready to be used and has a good starting list of packages on its official repository.

**Comment by saenai255** [comment]
Hi, yes, you are right that Pacstall is ready to be used and growing rapidly.

Regarding your suggestion, after the website is in a good shape, we plan to create our own *Pamac for Pacstall* kind of app. Integration with 3rd party software managers, like Gnome Software, is possible via plugins but the effort to maintain a plugin for each major software manager would be greater than just creating our own.
